packages/perspective-viewer-d3fc/src/js/series/seriesColors.js

```diff
@@ -45,3 +45,21 @@ export function withOpacity(color) {
     const toInt = offset => parseInt(color.substring(offset, offset + 2), 16);
     return `rgba(${toInt(1)},${toInt(3)},${toInt(5)},0.5)`;
 }
+
+export function seriesUpColors(domain) {
+    const range = domain.length <= 2 ? ["lightgreen", "green"] : d3.schemeGreens[domain.length];
```

Comment:
I'm sure there is a good reason but why not:
```suggestion
    const range = d3.schemeGreens[domain.length];
```
Same for down colours.

Reply:
Good question, that was my original solution, but unfortunately schemeGreens/schemeReds don't support domains of less than 3. I considered adding a comment explaining this, because I thought it wasn't obvious. I can do that if you think it would help.

Reply:
That's annoying, oh well. Up to you for the comment, I guess it wouldn't hurt.